sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -821,36 +821,6 @@ class HiveDDLSuite
       s"$tableName is a table. '$cmdName' expects a view. Please use ALTER TABLE instead.")
   }
 
-  test("create table - SET TBLPROPERTIES EXTERNAL to TRUE") {
```

Comment:
Instead of deleting the tests, can we ignore them instead please? I believe replacing `test` with `ignore` should do